Resources/Prototypes/SimpleStation14/Entities/Mobs/NPCs/primates.yml

```diff
@@ -0,0 +1,24 @@
+- type: entity
+  name: gorilla
+  parent: MobGorilla
+  suffix: Syndicate
+  id: MobGorillaSyndicate
+  description: Smashes, roars, looks cool. Don't stand near one.
+  components:
+  - type: Inventory
+    speciesId: syndimonkey
+    templateId: syndimonkey
+  - type: InventorySlots
+  - type: Loadout
+    prototypes: [ SyndimonkeyGear ]
+  - type: DraconicBloodstream
+    reagent:
+      solutionName: chemicals
+      reagent: Stimulants
+      unitsPerUpdate: 1.0
+      accumulatorTime: 5
```

Comment:
there's a better system for this now I believe, look at the trollospray

Reply:
yep I guess so